src/http.ts

```diff
@@ -143,7 +153,7 @@ class Http {
       }
 
       xhr.onerror = function() {
-        if (host && host === 'https://radar-verify.com:52516') {
+        if (host && (host === 'http://localhost:52516' || host === 'https://radar-verify.com:52516')) {
```

Comment:
We have like 3 different hard-coded URLs now, we should probably have them in a config or constants file somewhere:
```
https://api-verified.radar.io
https://radar-verify.com:52516
http://localhost:52516
```